client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/commands/SubmitCommand.kt

```diff
@@ -31,7 +36,8 @@ internal class SubmitCommand(private val submissionService: SubmissionService) :
             password = password,
             onBehalf = onBehalf,
             file = input,
-            attached = attached?.split(FILES_SEPARATOR)?.flatMap { getFiles(File(it)) }.orEmpty()
+            attached = attached?.split(FILES_SEPARATOR)?.flatMap { getFiles(File(it)) }.orEmpty(),
```

Comment:
I think we can move orEmpty to the begining to avoid the null chcecks 
`attached.orEmpty().split(FILES_SEPARATOR).flatMap { getFiles(File(it)) }`